src/modules/charging-import/lib/import-charge-version-metadata.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { logger } = require('../../../logger');
+const chargeVersionImportService = require('../services/charge-version-metadata-import');
+
+/**
+ * Run SQL queries to import charge versions / elements into
+ * water service tables from NALD import tables
+ * @return {Promise}
+ */
+const importChargeVersionMetadata = async () => {
+  try {
+    logger.info('Starting charge version metadata import');
+
+    await chargeVersionImportService.importChargeVersions();
```

Comment:
Super minor thing but I would suggest renaming `importChargeVersions` to `importChargeVersionsMetadata` to avoid confusion if the function is referenced in the future without its parent file.

Reply:
Done